### DISCUSSION TITLE
Lesson12: Error Running test. setUp failed reason="Index out of bounds" contract=0x7fa9385be102ac3eac297483dd6233d62b3e1496

### DISCUSSION THREAD

**Discussion by Glow-in-the-dark** [Cyfrin/foundry-full-course-cu#315] [Q&A] [answered]
I am trying to run my test, but met with this error. Not exactly sure how to solve it.
Any idea ?
![Screenshot 2023-07-12 at 3 05 28 AM](https://github.com/Cyfrin/foundry-full-course-f23/assets/2751458/4478ee42-d156-4765-8764-c7d15400b9e2)

This is my repo, if there is a need to replicate the error:
https://github.com/Glow-in-the-dark/Solidity-playground/tree/main/EtherJS_Foundry/DEFI-Stablecoins

Appreciate it and thanks!

**Comment by shikhar229169** [comment, marked as answer]
Hehe, found it.
Just replace that 1 by i.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/111875681/e32f67c4-1cd9-43d2-b35d-d7a3fe4ba6cf)

**Reply by Glow-in-the-dark** [reply]
ah dang! thanks so much.
